packages/js/src/types.ts

```diff
@@ -179,4 +179,8 @@ export type NovuOptions = {
   backendUrl?: string;
   socketUrl?: string;
   useCache?: boolean;
+  /**
+   * @internal Should be used internally
+   */
+  __userAgent?: string;
```

Comment:
couldn't find a better way to make it not expose to the public
Let me know if there's a better approach.
This `__userAgent` will appear in the autocomplete list if someone tries to do:

```js
 const novu = new Novu()
 ```